### PR TITLE
Mobile metrics: enable benchmarks

### PR DESCRIPTION
**Pull request by RingerJK**
### Description
Mobile metrics: enable

### CODE REVIEW

File: scripts/trigger-mobile-metrics.py [hunk 1]
```diff
@@ -75,6 +75,7 @@ def Main():
 
   # These jobs need to be refactored into workflows.
   if publishResults:
+    TriggerJob(token, commit, "android-navigation-benchmark")
```
Comment (RingerJK) [inline scripts/trigger-mobile-metrics.py:78]: cc @kmadsen

File: scripts/trigger-mobile-metrics.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def TriggerWorkflow(token, commit, publish):
       print("Started run_android_navigation_benchmark: %s" % response_dict)
 
 def TriggerJob(token, commit, job):
-    url = "https://circleci.com/api/v1.1/project/github/mapbox/mobile-metrics/tree/main"
```
Comment (kmadsen) [inline scripts/trigger-mobile-metrics.py:39]: nice catch :) looks like it was [find and replace all](https://github.com/mapbox/mapbox-navigation-android/commit/b8233660c7f09754f58c85874b3709fda499b26f)

File: scripts/trigger-mobile-metrics.py (hunk 1 above)
Comment (kmadsen) [reply scripts/trigger-mobile-metrics.py:78]: do you want to remove the TriggerJob's for `android-navigation-code-coverage` and `android-navigation-binary-size`. or are they working?

should keep the dry run ones though.

File: .circleci/config.yml [hunk 3]
```diff
@@ -526,7 +526,7 @@ jobs:
           artifact-type: "core"
       - publish-artifacts:
           artifact-type: "ui"
-#      - trigger-mobile-metrics
```
Comment (kmadsen) [inline .circleci/config.yml:529]: Let's verify that this runs before enabling the production run

Screen Shot 2021-05-20 at 8 23 47 AM

File: scripts/trigger-mobile-metrics.py (hunk 1 above)
Comment (RingerJK) [reply scripts/trigger-mobile-metrics.py:78]: yeah, wanted to sync. Think this's a good place to disable these jobs 👍

Comment (kmadsen) [reply scripts/trigger-mobile-metrics.py:78]: yeah just comment out the publishResults lines
```
    TriggerJob(token, commit, "android-navigation-code-coverage")
    TriggerJob(token, commit, "android-navigation-binary-size")
```

but keep the triggers in the else block, so we can continue to test them

**Review by kmadsen — APPROVED** [review]

**Pull request merged**